src/gui/dispersionreticle/translations/translations_zh_cn.json

```diff
@@ -35,33 +35,33 @@
   "focusedReticle.header": "最小落弹环",
   "focusedReticle.body": [
     "在显示标准瞄准环的同时，显示出最小的缩圈大小。\n",
-    "Uses builtin game logic to do so.\n",
-    "Other mods might influence the look of this reticle.\n"
+    "使用游戏机制实现。\n",
+    "同时使用其他类似模组可能影响显示效果。\n"
   ],
   "focusedReticle.note": [
     "在打开了插件中的服务器准星选项时最小缩圈将显示在客户端准星上。\n\n"
   ],
-  "focusedReticleExtended.label": "Focused reticle extended",
-  "focusedReticleExtended.header": "Focused reticle extended",
+  "focusedReticleExtended.label": "最小落弹环外观配置",
+  "focusedReticleExtended.header": "最小落弹环外观配置",
   "focusedReticleExtended.body": [
-    "Adds configurable reticle displaying fully-focused dispersion to in-game reticle.\n",
-    "Other mods does not have influence on the look of this reticle."
+    "这里可以配置最小落弹环的外观。\n",
+    "同时使用其他类似模组可能影响显示效果。"
   ],
   "focusedReticleExtended.note": [
-    "When both client-side and server-side reticles are on, it attaches to client-side reticle."
+    "当同时启用了模组中的客户端准星和服务端准星时，最小落弹环将出现在客客户端准星上。"
   ],
   "focusedReticleExtended.attention": [
-    "For SPG artillery view, a standard version of this reticle will be used."
+    "火炮视角中最小落弹环将按游戏默认形式显示。"
   ],
   "hybridReticle.label": "延迟瞄准环",
```

Comment:
Latency reticles are now named hybrid reticles (if there is some translation for it), but latency translation can be left as it is if you want